torch/nn/modules/loss.py

```diff
@@ -1471,7 +1475,7 @@ class TripletMarginLoss(_Loss):
 
     Examples::
 
-    >>> triplet_loss = nn.TripletMarginLoss(margin=1.0, p=2)
+    >>> triplet_loss = nn.TripletMarginLoss(margin=1.0, p=2, eps=1e-6)
```

Comment:
nit: maybe use a different value from the default :)

Reply:
Thanks @mikaylagawarecki for your feedback. I was considering changing the `eps` value to `1e-7` in the example. Would that be appropriate, or do you suggest another value?